### PR TITLE
Feature/#17 lib changes for darkmode and link improvements

### PR DESCRIPTION
**Pull request by rudigier**

### CODE REVIEW

File: src/components/Footer.tsx
Comment (flxtagi) [inline src/components/Footer.tsx:1]: 🦶  ! Nice

File: src/components/ThemeSwitch.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { useThemeContext, THEME } from '../context/useTheme';
```
Comment (flxtagi) [inline src/components/ThemeSwitch.tsx:1]: I am not sure about the our namiing convention is happy whithe all caps THEME...

File: src/components/UserRecommender.tsx [hunk 2]
```diff
@@ -7,35 +7,36 @@ import { services } from '../services';
...
 	const accessToken = session?.accessToken;
 
-	// its clientside and its working better IMHO
 	useEffect(() => {
 		if (accessToken) {
-			const fetchRecommendedUsers = async (accessToken: string) => {
+			const fetchRecommendedUsers = async () => {
 				const recommendedUsers = await services.users.getUsers({
-					limit: 7,
+					limit: 100, // TODO: this is a workaround, because the API does not support randomization
```
Comment (flxtagi) [inline src/components/UserRecommender.tsx:24]: maybe 50 would be enough for now.. but it works too.

File: tailwind.config.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 /** @type {import('tailwindcss').Config} */
 
 module.exports = {
+	darkMode: 'class',
```
Comment (flxtagi) [inline tailwind.config.js:4]: nice ! one class ;-)

**Review by flxtagi — APPROVED** [review]
I need this update! also TS version 5  now. great. 
tested on my machine. dark mode and footer is a nice addon. 
May we put the dark-mode switcher into the user-settings, once that set up propperly ?

File: src/components/UserRecommender.tsx (hunk 2 above)
Comment (rudigier) [reply src/components/UserRecommender.tsx:24]: Doesn't matter for now. It's just a workaround for a missing functionality.

**Pull request merged**